rosserial_python/src/rosserial_python/SerialClient.py

```diff
@@ -311,23 +312,28 @@ def inWaiting(self):
             if chunk == '':
```

Comment:
This should be `if chunk == b'':` I think.

Reply:
Agree, changed.